src/Fmt_ast.ml

```diff
@@ -357,6 +357,10 @@ let fmt_docstring ?pro ?epi doc =
              $ Option.call ~f:epi ) ) )
 
 
+let fmt_keyword_with_extension keyword ext =
+  str keyword $ opt ext (fun {txt; loc} -> str "%" $ Cmts.fmt loc (str txt))
```

Comment:
I'm not sure if it is simpler to include `str keyword $` in this function, or if it might be better to shorten this to a function that just formats `ext`. Maybe consider both and see what you think.